app/src/main/java/org/wikipedia/events/ArticleSavedOrDeletedEvent.kt

```diff
@@ -0,0 +1,7 @@
+package org.wikipedia.events
+
+import org.wikipedia.readinglist.database.ReadingListPage
+
+class ArticleSavedOrDeletedEvent(val isAdded: Boolean, vararg pages: ReadingListPage) {
+    val pages: Array<ReadingListPage> = arrayOf(*pages)
```

Comment:
This can be simplified as `val pages = pages.toList()`

Reply:
Sure thing!